src/ProofTools/DeiterationTool.ts

```diff
@@ -30,7 +30,10 @@ let currentProofTree: AEGTree;
  */
 export function deiterationMouseDown(event: MouseEvent) {
     const currentPoint: Point = new Point(event.x - offset.x, event.y - offset.y);
-    currentProofTree = new AEGTree(treeContext.getLastProofStep().tree.sheet);
+    currentProofTree = new AEGTree();
+    if (treeContext.currentProofStep) {
+        currentProofTree.sheet = treeContext.currentProofStep.tree.sheet.copy();
+    }

```

Comment:
If we need this exact block of code in every single proof mode tool it should be turned into a helper function.

Reply:
Still holds

Reply:
This has been added to the master clean up issue.